introduction.rst

```diff
@@ -0,0 +1,43 @@
+############
+Introduction
+############
+
+Welcome to Network4Dev! Network4Dev is a community-driven, open source 
+networking site. The site was founded by 
+`Daniel Dib <https://www.linkedin.com/in/danieldib/>`_ in 2018 and all 
+work behind the scenes is done by 
+`Cristian Sirbu <https://www.linkedin.com/in/sirbucristian/>`_. At the
```

Comment:
I'd prefer the link to go to my website's about page (https://www.trueneutral.eu/pages/about.html) as linkedin starts hiding public profiles if they get more than a few views.

Reply:
Link updated.